.github/workflows/build.yaml

```diff
@@ -58,3 +58,39 @@ jobs:
       - name: Build MySQL extension
         if: ${{ matrix.php.major < 7 }}
         run: nix-build -A outputs.packages.x86_64-linux.php${{ matrix.php.major }}${{ matrix.php.minor }}.extensions.mysql
+
+      - name: Validate php.extensions.mysqli default unix socket path
+        if: ${{ matrix.php.major < 7 }}
+        run: |
+          nix-shell -E '
+            let
+              self = import ./.;
+              php =
+                self.outputs.packages.${builtins.currentSystem}.php56.withExtensions
```

Comment:
This should use the versions from the matrix.

Reply:
OK. I'll remove the `if` statement as well so it runs on all versions.